core/templates/pages/skill-editor-page/skill-editor-page.component.html

```diff
@@ -77,8 +77,10 @@
     margin-left: 4%;
     width: 35%;
   }
-  .skill-editor-main .oppia-editor-warnings-box-block {
-    display: block;
+
+  .skill-editor-main .oppia-editor-warnings-box {
+    cursor: default;
+    max-width: none;
```

Comment:
Why is this rule required?

Reply:
Actually, I'm trying to replicate the issue on the skill editor (which doesn't seem to be showing for some reason). I saw that the class in the styling which was mentioned previously didn't match the styling which was actually given in the HTML file, so I changed it. It employs a similar style as the warnings box class for the other editors. (The cursor property solves the issue with the error being highlighted).

Reply:
If I can replicate the issue on the skill editor, I'll push fresh changes to the PR and fix it. (and also add a screenshot).